t/50access-log-transport.t

```diff
@@ -0,0 +1,98 @@
+use strict;
+use warnings;
+use Net::EmptyPort qw(empty_port wait_port);
+use File::Temp qw(tempdir);
+use JSON;
+use Test::More;
+use t::Util;
+
+my $tempdir = tempdir(CLEANUP => 1);
+
+my $client_prog = bindir() . "/h2o-httpclient";
+plan skip_all => "$client_prog not found"
+    unless -e $client_prog;
+
+my $quic_port = empty_port({
+    host  => "127.0.0.1",
+    proto => "udp",
+});
+
+my $conf = << "EOT";
+access-log:
+    format: '{"protocol":"%H","cc.name":"%{cc.name}x","delivery_rate":"%{delivery-rate}x"}'
+    escape: json
+    path: "$tempdir/access_log"
+listen:
+  type: quic
+  port: $quic_port
+  ssl:
+    key-file: examples/h2o/server.key
+    certificate-file: examples/h2o/server.crt
+  cc: pico
```

Comment:
I'm not sure if we need this. It is the task of the transport logic of quicly to estimate the delivery rate. Not the particular CC implementation.

Reply:
Removed!

I added it because I was finding a way to test the CC algo for other tests. Now it was irrelevant to the delivery rate, so I just removed it.